src/app/public/modules/media-query/media-query.service.ts

```diff
@@ -37,10 +37,10 @@ export class SkyMediaQueryService {
   private mdMql: MediaQueryList;
   private lgMql: MediaQueryList;
 
-  private xsListener: MediaQueryListListener;
-  private smListener: MediaQueryListListener;
-  private mdListener: MediaQueryListListener;
-  private lgListener: MediaQueryListListener;
+  private xsListener: any;
```

Comment:
Why are we changing these? Properties which are `any` in main code classes make me nervous :)

Reply:
The type `MediaQueryListListener` was removed in TypeScript 3.1 (https://stackoverflow.com/questions/53010349/using-mediaquerylistlistener-in-typescript-3-1), and the type itself isn't well supported (https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryListListener). Ideally, we would refactor this service to work with "approved" types, but that will take some doing.

Reply:
Ok. Sounds good.